From 3bb0280cadda234532e4ca3823894c3d400da1b6 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 5 Feb 2025 17:03:33 +0100
Subject: [PATCH 1586/1695] dts: stm32: drop wakeup-parent from RTC nodes

The property is not used anymore.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d8206fac2b720fcf7edcbef1f80205b049afa2b9 dts: stm32: drop wakeup-parent from RTC nodes]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ibb1bb671fa3fef6e20bad45c7a01816ef5a2fdc1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/434932
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 1 -
 core/arch/arm/dts/stm32mp251.dtsi | 1 -
 2 files changed, 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 5aedfd78e..2520ac962 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -1130,7 +1130,6 @@
 			clocks = <&rcc CK_BUS_RTC>, <&rcc RTC_CK>;
 			clock-names = "pclk", "rtc_ck";
 			wakeup-source;
-			wakeup-parent = <&exti2 0xffff 0>;
 			interrupts-extended = <&exti2 22 IRQ_TYPE_EDGE_RISING>;
 		};
 
diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index d6b337228..cd699ad57 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -1217,7 +1217,6 @@
 			clocks = <&rcc CK_BUS_RTC>, <&rcc RTC_CK>;
 			clock-names = "pclk", "rtc_ck";
 			wakeup-source;
-			wakeup-parent = <&exti2 0xffff 0>;
 			interrupts-extended = <&exti2 22 IRQ_TYPE_EDGE_RISING>;
 		};
 
-- 
2.47.2

